api/src/main/java/io/opentelemetry/trace/TraceId.java

```diff
@@ -84,8 +84,9 @@ public static TraceId getInvalid() {
    *
    * @param random the random number generator.
    * @return a new valid {@code TraceId}.
+   * @since 0.1.0
    */
-  static TraceId generateRandomId(Random random) {
+  public static TraceId generateRandomId(Random random) {
```

Comment:
Same here.

Reply:
I have moved ID generation methods to SpanBuilder in SDK